hooks/VDM.js

```diff
@@ -305,303 +305,229 @@ module.exports = function (input, session) {
   }
 
   if (data.lon && data.lat) {
-    meteoLocation = data.lon.toString().replace('.', '_') + "__" + data.lat.toString().replace('.', '_')
+    values.push({
+      path: 'navigation.position',
+      value: {
+        longitude: data.lon,
+        latitude: data.lat,
+      },
+    })
   }
 
   if (data.avgwindspd) {
     contextPrefix = 'meteo.'
-    if (data.avgwindspd < 127) {
-      values.push({
-        path: `environment.observations.${meteoLocation}.air.windAverageSpeed`,
-        value: utils.transform(data.avgwindspd, 'knots', 'ms'),
-      })
-    } 
+    values.push({
+      path: `environment.air.windAverageSpeed`,
+      value: utils.transform(data.avgwindspd, 'knots', 'ms'),
+    })
   }
 
   if (data.windgust) {
     contextPrefix = 'meteo.'
-    if (data.windgust < 127) {
-      values.push({
-        path: `environment.observations.${meteoLocation}.air.windGust`,
-        value: utils.transform(data.windgust, 'knots', 'ms'),
-      })
-    }
+    values.push({
+      path: `environment.air.windGust`,
```

Comment:
Use `environment.observations.wind.xxx` instead of `environment.observations.air`.